phasmophobia_evidence_v3/widget.js

```diff
@@ -715,6 +886,77 @@ let getOptObj = (obj) => {
   return optObj;
 }
 
+let setMapName = (name) => {
+  let map = ""
+  
+  let nameSplit = name.split(' ');
+  
+  switch (nameSplit[0].toLowerCase()) {
+    case 'ta':
+    case 'tangle':
+    case 'tanglewood':
+      map = "Tanglewood Street House"
+      break;
+    case 'ed':
+    case 'edge':
+    case 'edgefield':
+      map = "Edgefield Street House"
+      break;
+    case 'ri':
+    case 'ridge':
+    case 'ridgeview':
+      map = "Ridgeview Road House"
+      break;
+    case 'gr':
+    case 'grafton':
+      map = "Grafton Farmhouse"
+      break;
+    case 'bl':
+    case 'bleasdale':
+      map = "Bleasdale Farmhouse"
+      break;
+    case 'hi':
+    case 'high':
+    case 'school':
+    case 'brown':
+    case 'brownstone':
+      map = "Brownstone High School"
+      break;
+    case 'pr':
+    case 'prison':
+      map = "Prison"
+      break;
+    case 'as':
+    case 'asylum':
+      map = "Asylum"
+      break;
+  }
+  $('#map-name').html(map);
+  
+  if (nameSplit[1]) {
+    setMapDiff(nameSplit[1]);
+  }
+}
+
+let setMapDiff = (name) => {
+  let diff = ""
+  switch (name.toLowerCase()) {
```

Comment:
Add cases for single letter abbreviations `a, i, p`

Reply:
This might get a bit tricky with new maps on the (eventual) horizon. I wouldn't want people to get used to a for asylum and then all of a sudden there's an "Abbeyville Apartments" or something silly. Thoughts?

I do know that Glitch is looking into updating the system for cases like optional objectives, ghost types, etc. so that the code doesn't have to be edited in the future and it's easier for him/us when DK adds things.

Reply:
This comment was meant for Difficulty, not Map.

Reply:
The abbreviations I was referring to were for the difficulty. a = amateur, i = intermediate, p = professional.